utest/utils/test_misc.py

```diff
@@ -1,7 +1,11 @@
 import unittest
 
-from robot.utils.misc import getdoc, printable_name, seq2str
-from robot.utils.asserts import assert_equals
+import sys
+import io
+
+from robot.utils.misc import getdoc, printable_name, seq2str, isatty
```

Comment:
Even more nitpicking: We prefer organizing imports alphabetically i.e. `isatty` should be after `getdoc`. Clearly this file doesn't fully follow that rule otherwise either, because `robot.utils.asserts` line should be before `robot.utils.misc`.